lib/transaction/tracecontext.js

```diff
@@ -48,9 +48,14 @@ class TraceContext {
   }
 
   /**
+   * @param spanContext
```

Comment:
Each of the docblock updates in this file is missing the required "type" field:
```suggestion
   * @param {object} spanContext
```

Reply:
I'll clean up.